src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -155,6 +157,33 @@ export CONDUCTR_HAPROXY_CONFIG_OVERRIDE="$CONFIG_DIR/haproxy-override.cfg"
 The `CONFIG_DIR` refers to the directory where the configuration override will be expanded. `CONDUCTR_HAPROXY_CONFIG_OVERRIDE` is the environment variable which tells ConductR where to find the custom HAProxy configuration template.
 
 
+### Specifying custom HAProxy reload script
+
+ConductR HAProxy makes use of the reload script to reload HAProxy whenever new HAProxy configuration is generated. When ConductR HAProxy starts, the reload script is copied from the path specified by `HAPROXY_RELOAD_SCRIPT_SOURCE` environment variable to the path specified by `HAPROXY_RELOAD_SCRIPT_LOCATION` environment variable.
```

Comment:
Also, stating '_the_ reload script' implies that the reader knows about the script. I don't think it has been introduced yet here. I think 'a reload script' is better in the first sentence of this paragraph.